docs_src/pipelinerun.md

```diff
@@ -41,36 +41,59 @@ Note: there could be multiple containers with IMAGE "openvino/ubuntu20_data_runt
 ## Run pipeline with different input source(inputsrc) types
 Use docker-run.sh to run the pipeline
 
-### option 1 to run object detection, object classification, OCR, and barcode recognition with simulated camera:
+### Option 1 to run object detection, object classification, OCR, and barcode recognition with simulated camera:
 
+#### For Core systems
+```
+sudo ./docker-run.sh --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0
 ```
-./docker-run.sh --platform core|xeon|dgpu.x --inputsrc rtsp://127.0.0.1:8554/camera_0 --ocr 5 GPU
-```  
 
-### option 2 to run object detection with simulated camera:
+#### For Xeon systems
+```
+sudo CPU_ONLY=1 ./docker-run.sh --platform xeon --inputsrc rtsp://127.0.0.1:8554/camera_0
+```
 
+#### For Discrete GPU systems
 ```
-./docker-run.sh --platform core|xeon|dgpu.x --inputsrc rtsp://127.0.0.1:8554/camera_0 --classification_disabled --ocr_disabled --barcode_disabled
+sudo ./docker-run.sh --platform dgpu.x --inputsrc rtsp://127.0.0.1:8554/camera_0 --ocr 5 GPU
 ```  
+Note: for `dgpu.x`, the x can be 0, 1, 2 depends on how many discrete gpu it has in the system
 
-### option 3 to run object detection, object classification, OCR, and barcode recognition with USB Camera:
 
+### Option 2 to run object detection, object classification, OCR, and barcode recognition with USB Camera:
+
+#### For Core systems
 ```
-./docker-run.sh --platform core|xeon|dgpu.x --inputsrc /dev/video0
-``` 
+sudo ./docker-run.sh --platform core --inputsrc /dev/video0
+```
+Query USB camera: [How_to_query_usb_camera](./query_usb_camera.md)
 
-### option 4 to run with RealSense Camera(serial number input):
+### Option 3 to run with RealSense Camera(serial number input):
 
+#### For Core systems
 ```
-./docker-run.sh --platform core|xeon|dgpu.x --inputsrc serial_number --ocr 5 GPU --realsense_enabled
+sudo ./docker-run.sh --platform core --inputsrc <serial_number> --realsense_enabled
 
 ```
 Obtaining RealSense camera serial number: [How_to_get_serial_number](./camera_serial_number.md)
-### option 5 to run with video file input:
 
+### Option 4 to run with video file input:
+
+#### For Core systems
 ```
-./docker-run.sh --platform core|xeon|dgpu.x --inputsrc file:my_video_file.mp4 --ocr 5 GPU
-``` 
+sudo ./docker-run.sh --platform core --inputsrc file:my_video_file.mp4
+```
+
+#### For Xeon systems
+```
+sudo CPU_ONLY=1 ./docker-run.sh --platform xeon --inputsrc file:my_video_file.mp4
```

Comment:
CPU_ONLY is not needed when specifying --platform xeon. CPU_ONLY is only needed for platforms (like Core) where the SOC has a CPU and iGPU and we want use CPU vs. (iGPU or iGPU+CPU) pipelines

Reply:
we got that CPU_ONLY based on the test plan of benckmarking for walserv3 to test it on

Reply:
yeah, we've got CPU_ONLY from benchmarking test plan. I will test it on walserv3 again without CPU_ONLY, I can remove it if it's working on walserv3

Reply:
Don't see the change but approving based on comment.

Reply:
I thought that if you did not have CPU_ONLY and there is a dgpu in the system (which walserv3 has), docker run will run the dgpu pipeline instead of xeon. Unless that logic has changed.

Reply:
That was not the intention. Take a look at https://github.com/intel-retail/vision-self-checkout/blob/main/configs/framework-pipelines/core/yolov5s_effnetb0.sh. The optimized pipeline will default to CPU+iGPU but we can override to pure iGPU or CPU